<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SMExtendTaskPanel</class>
 <widget class="QDialog" name="SMExtendTaskPanel">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>259</width>
    <height>560</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Extend properties</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <property name="topMargin">
      <number>0</number>
     </property>
     <item>
      <widget class="QPushButton" name="AddRemove">
       <property name="text">
        <string>Select</string>
       </property>
       <property name="checkable">
        <bool>true</bool>
       </property>
       <property name="checked">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushClearSel">
       <property name="text">
        <string>Clear selection</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QTreeWidget" name="tree">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="headerHidden">
      <bool>true</bool>
     </property>
     <column>
      <property name="text">
       <string>Name</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>SubElement</string>
      </property>
     </column>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupExtend">
     <property name="title">
      <string>Extend</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="0" column="1">
       <widget class="Gui::QuantitySpinBox" name="Length" native="true">
        <property name="unit" stdset="0">
         <string notr="true">mm</string>
        </property>
        <property name="minimum" stdset="0">
         <double>0.000000000000000</double>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="Gui::QuantitySpinBox" name="OffsetB" native="true">
        <property name="unit" stdset="0">
         <string notr="true">mm</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="Gui::QuantitySpinBox" name="OffsetA" native="true">
        <property name="unit" stdset="0">
         <string notr="true">mm</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="label_5">
        <property name="sizePolicy">
         <sizepolicy hsizetype="MinimumExpanding" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string>Offset B</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label_4">
        <property name="text">
         <string>Offset A</string>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>Length</string>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QCheckBox" name="checkReversed">
        <property name="text">
         <string>Reversed</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupExtendBySketch">
     <property name="title">
      <string>Extend by sketch</string>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="0">
       <widget class="QPushButton" name="buttSelSketch">
        <property name="text">
         <string>Extend Sketch...</string>
        </property>
        <property name="checkable">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLineEdit" name="txtSelSketch">
        <property name="readOnly">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QCheckBox" name="checkIntersectClear">
        <property name="text">
         <string>Intersect clearance</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="Gui::QuantitySpinBox" name="Offset" native="true">
        <property name="unit" stdset="0">
         <string notr="true">mm</string>
        </property>
        <property name="minimum" stdset="0">
         <double>0.000000000000000</double>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QPushButton" name="buttClearSketch">
        <property name="text">
         <string>Clear Sketch</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QCheckBox" name="RefineCheckbox">
     <property name="text">
      <string>Refine</string>
     </property>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>Gui::QuantitySpinBox</class>
   <extends>QWidget</extends>
   <header>Gui/QuantitySpinBox.h</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>tree</tabstop>
  <tabstop>Length</tabstop>
  <tabstop>OffsetA</tabstop>
  <tabstop>OffsetB</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
